src/components/Activities/Activity1/Game1/components/CipherGameRound.tsx

```diff
@@ -0,0 +1,119 @@
+
+import { scramble } from '../../../../../utils';
+import Alien, { ALIEN_STATE } from '../../../../shared/Alien';
+import { TextBubbleStyles, Activity1Game1Values } from '../../../../shared/PlaynetConstants';
+
+import TextBubble from '../../TextBubble';
+import CipherGameSlide from './../components/CipherGameSlide';
+import ProgressBar from './../components/ProgressBar';
+
+export interface SlideComponents {
+  correctIdx: number,
+  cards: string[],
+}
+
+interface CipherGameRoundProps {
+  round: string[];
+  advanceGame: () => void;
+  HASH_VAL: number;
+}
+
+function CipherGameRound(props : CipherGameRoundProps): JSX.Element {
+  const {MAX_HAPPINESS, CORRECT_PTS, INCORRECT_PTS} = Activity1Game1Values;
+  const {round, advanceGame, HASH_VAL} = props;
+
+  const getShuffledCards = () => {
+    let cards = [...round];
+    const newSlides = [];
+    for (let i = 0; i < 3; i++) {
+      if (cards.length === 1) cards = [...round];
+      const card0 = cards[Math.floor(Math.random()*cards.length)]; // get card #1
+      cards = cards.filter(item => item !== card0);
+      const card1 = cards[Math.floor(Math.random()*cards.length)]; // get a non duplicate card
+
+      if (Math.floor(Math.random()*2)) { // want correct img to be card #1
+        newSlides.push(
+          {
+            correctIdx: 0,
+            cards: [card0, card1],
+          },
+        );
+      } else {
+        newSlides.push(
+          {
+            correctIdx: 1,
+            cards: [card1, card0],
+          },
+        );
+      }
+    }
+    return newSlides;
+  };
+
+  const [slides, setSlides] = useState<SlideComponents[]>(getShuffledCards());
+  const [slideIdx, setSlideIdx] = useState(0);
+  const [happiness, setHappiness] = useState(0);
+
+  // -----ALIEN HANDLERS------
+  const [alienState, setAlienState] = useState(ALIEN_STATE.BASE);
+  const alienTimeout = useRef<NodeJS.Timeout | undefined>(undefined);
+
+  const handleAlienState = (state: ALIEN_STATE, cb?: () => void) => {
+    alienTimeout.current && clearTimeout(alienTimeout.current);
+    setAlienState(state);
+    alienTimeout.current = setTimeout(() => {
+      alienTimeout.current = undefined;
+      setAlienState(ALIEN_STATE.BASE);
+      cb && cb();
+    }, 1000);
+  };
+  // ------------------------
+  const displayScrambledText = () : string => { // split phrase into words and scramble individually
+    const slide = slides[slideIdx];
+    const correctWord = slide.cards[slide.correctIdx].split(' ');
+    const reducer = (acc : string, cur : string) => `${acc} ${scramble(HASH_VAL, cur)}`;
+    return correctWord.reduce(reducer, '');
+  };
+
+  const advanceRound = (correct : boolean) => {
+    if (correct) {
+      const newHappiness = Math.min(happiness+CORRECT_PTS, 100); // no overflow
+      const handler = newHappiness === MAX_HAPPINESS ? advanceGame : () => {return;}; // prevent no-op by finishing animation before scene change
```

Comment:
nit: i think its a bit easier to remember the card if you see it on the screen a bit?
```suggestion
      const handler = newHappiness === MAX_HAPPINESS ? advanceGame : () => nextSlide(); // prevent no-op by finishing animation before scene change
```